### PR TITLE
Content Curators README.md fixes

### PR DESCRIPTION
**Pull request by kek-mex**
Content Curators README.md fixes

### CODE REVIEW

File: roles/content-curators/README.md [hunk 1]
```diff
@@ -102,8 +102,8 @@ The main job for curators is to regularly check the content directory and channe
 ## Content
 
 Curators are responsible for ensuring the following:
-1. Content are not in violation with the platform [ToS](https://testnet.joystream.org/#/pages/tos), meaning
-  - illicit should be `censored`, and marked for takedown
+1. Content is not in violation with the platform [ToS](https://testnet.joystream.org/#/pages/tos), meaning
+  - illicit should be `censored`, and marked for a takedown
```
Comment (kek-mex) [inline roles/content-curators/README.md]: Maybe "content" should be added here, resulting in the following: "illicit content should be `censored`, and marked for a takedown"?

**Comment by kek-mex** [comment]
@blrhc

File: roles/content-curators/README.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ This page will contain all information on how to apply for the role of `Content
 
 # Hiring Process
 
-First, open [Pioneer](testnet.joystream.org/), and navigate to the `Working groups` sidebar. Here you will find an overview of the current `Content Curators`, and the `Content Lead`. In the `Opportunities` tab, you can get an overview of future, current and previous openings, along with terms, settings, status and history (if applicable). The terms and settings include the following parameters to consider:
+First, open [Pioneer](testnet.joystream.org/), and navigate to the `Working groups` sidebar. Here you will find an overview of the current `Content Curators`, and the `Content Lead`. In the `Opportunities` tab, you can get an overview of future, current and previous openings, along with terms, settings, status, and history (if applicable). The terms and settings include the following parameters to consider:
```
Comment (mnaamani) [inline roles/content-curators/README.md]:
```suggestion
First, open [Pioneer](https://testnet.joystream.org/), and navigate to the `Working groups` sidebar. Here you will find an overview of the current `Content Curators`, and the `Content Lead`. In the `Opportunities` tab, you can get an overview of future, current and previous openings, along with terms, settings, status, and history (if applicable). The terms and settings include the following parameters to consider:
```

File: roles/content-curators/README.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ These rules are not clearly defined in all cases, so it's important that curator
 
 # Advanced
 
-Some of the tasks expected by curators require manual [extrinsics](https://testnet.joystream.org/#/extrinsics). Not  all required tasks can be done in the UI directly, but they can all be done through extrinsics.
+Some of the tasks expected by curators require manual [extrinsics](https://testnet.joystream.org/#/extrinsics). Not all required tasks can be done in the UI directly, but they can all be done through extrinsics.
```
Comment (mnaamani) [inline roles/content-curators/README.md]:
```suggestion
Some of the tasks expected of curators require manual [extrinsics](https://testnet.joystream.org/#/extrinsics). Not all required tasks can be done in the UI directly, but they can all be done through extrinsics.
```

**Pull request merged**